### PR TITLE
test: add test coverage - escalation boundary events migration

### LINKED ISSUE #24628: I can migrate escalation boundary events
### User Story

```Gherkin
Scenario: As an Operate user
    Given I am in instance migration view
    Then I'm able to map tasks inside sub processes with an escalation boundary event
    When I start the migration
    Then task is getting migrated to the new process definition version
    Then the escalation code of the target process definition version is set as escalation code
```

### Acceptance Criteria

- [x] I can map tasks inside sub processes with an escalation boundary event
- [x] I can not map escalation boundary events itself
- [x] The escalation code is updated to the target escalation code

### Definition of Ready - Checklist

- [x] The issue has a meaningful title, description, and testable acceptance criteria
- [ ] If documentation needs to be updated, an issue is created in the [camunda-docs](https://github.com/camunda/camunda-docs) repo, and the issue is added to our Operate project board.
- [ ] If HELM charts need to be updated, an issue is created in the [camunda-platform-heml](https://github.com/camunda/camunda-platform-helm) repo, and the issue is added to our Operate project board.
- [ ] Potential Security risks have been considered and added to the product-hub issue risk assessment
- [ ] Cross-team dependencies have been considered

Optional:
- [ ] Design input has been collected by the assignee
### :point_right: Handover Dev to QA 

- Handy resources: 
     BPMN/DMN models, plugins, scripts, REST API endpoints + example payload, etc : 
     
- Example projects:

- Versions to validate:
> Docker file : in case that it needed to be tested via docker share the version contain the fixed along with version of other services . 

- Release version ( in which version this fixed/feature will be released)

### PR DESCRIPTION
**Pull request by pedesen**
## Description

This PR adds E2E test coverage to ensure the migration works.
Mapping tasks inside sub processes with an escalation boundary event was already possible. 

The layout in the process definition files was updated due to readability reasons, which caused the big diff

Before:

image

After:

image

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #24628 

Link to the test case: 
[User Can Migrate successfully a process with escalation boundary events](https://camunda.testrail.com/index.php?/cases/view/535967)

### CODE REVIEW

File: operate/client/e2e-playwright/tests/processInstanceMigration.spec.ts [hunk 1]
```diff
@@ -142,10 +145,10 @@ test.beforeAll(async ({request}) => {
       },
       {timeout: SETUP_WAITING_TIME},
     )
-    .toHaveProperty('total', 10);
+    .toHaveProperty('total', totalInstances);
 });
 
-test.describe.serial('Process Instance Migration', () => {
+test.describe.serial.only('Process Instance Migration', () => {
```
Comment (vitorwtavares) [inline operate/client/e2e-playwright/tests/processInstanceMigration.spec.ts:151]: I believe this only is breaking the CI 😅

**Review by vitorwtavares — APPROVED** [review]

**Comment by pedesen** [comment]
@mschoe Let me know if you have any questions. Here's a docker image of the branch you can use for testing: `registry.camunda.cloud/team-operate/camunda-operate:branch-fe-24628-migrate-escalation-boundary`.

**Review by ArezooAzar — APPROVED** [review]

**Comment by ArezooAzar** [comment]
@pedesen : can you please review the test case [here](https://camunda.testrail.com/index.php?/cases/view/535967)

**Comment by pedesen** [comment]
> @pedesen : can you please review the test case [here](https://camunda.testrail.com/index.php?/cases/view/535967)

@ArezooAzar I updated the instructions in the last step: added the zbctl command and described more precisely what's expected to happen. Looks good otherwise!

**Pull request merged**